web/src/containers/admin/MyAccount.js

```diff
@@ -0,0 +1,165 @@
+import { Alert, Button, Spinner, TextField } from '@cmsgov/design-system-core';
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { editSelf } from '../../actions/admin';
+import Password from '../../components/PasswordWithMeter';
+import { t } from '../../i18n';
+
+class MyAccount extends Component {
+  state = {
+    success: false,
+    user: {
+      name: '',
+      password: '',
+      phone: '',
+      position: ''
+    }
+  };
+
+  constructor(props) {
+    super(props);
+
+    const {
+      user: { name, phone, position }
+    } = props;
+
+    this.state.user = { name, phone, position };
+  }
+
+  static getDerivedStateFromProps(newProps, prevState) {
+    if (newProps.fetching) {
+      return { success: 0 };
+    }
+    if (prevState.success === 0) {
+      return { success: !newProps.error };
+    }
+    return null;
+  }
+
+  handleEditAccount = e => {
+    const { name, value } = e.target;
+    this.setState(prev => ({ user: { ...prev.user, [name]: value } }));
+  };
+
+  editAccount = e => {
+    e.preventDefault();
+    const { editAccount } = this.props;
+    const { user } = this.state;
+    editAccount(user);
+  };
+
+  goBack = () => {
+    const {
+      history: { goBack }
+    } = this.props;
+    goBack();
+  };
+
+  render() {
+    const { fetching } = this.props;
+    const {
+      success,
+      user: { name, password, phone, position }
+    } = this.state;
+
+    return (
+      <Fragment>
+        <header className="clearfix mb2 px2 py1 bg-white">
+          <div className="left">
+            <Link to="/" className="btn px0 bold caps">
+              {t('titleBasic')}
+            </Link>
+          </div>
+        </header>
+
+        <div className="ds-l-col--5 ds-u-fill--white ds-u-margin-x--auto ds-u-margin-y--3 ds-u-padding--6 ds-u-radius">
+          <div className="ds-u-margin-x--auto" style={{ maxWidth: '460px' }}>
```

Comment:
This max width can go away once we put the `ds-l-col--6` class on.

Reply:
Is that because the max width on the inputs are also based on columns?  Or at least based on column widths, anyway.  That's what drove this `maxWidth`, but I love the direction this is going.

Reply:
The inputs should be inheriting a max width from the design system!